app/controllers/api/v1/admin/site_settings_controller.rb

```diff
@@ -23,6 +23,7 @@ def update
 
           update = if params[:name] == 'BrandingImage'
                      site_setting.image.attach params[:site_setting][:value]
+                     site_setting.update(value: site_setting.image.blob.filename.to_s)
```

Comment:
Um, I understand what you're trying to do.
It feels a bit hackery IMO, for now it's okay but I think that the `site_settings` fetching logic may require having its own service managing such cases,

Reply:
Yeah agreed.
I wanted to add the feature with the current mechanics.